os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -329,35 +332,29 @@ static int binary_manager_reload(char *bin_name)
 static int loading_thread(int argc, char *argv[])
 {
 	int ret;
-	int type;
+	loading_data_t *loading_data;
 
-	if (argc <= 1) {
-		bmdbg("Invalid arguments\n");
-		return -1;
-	}
-
-	/* Arguments : [1] type */
-	type = (int)atoi(argv[1]);
-
-	ret = BINMGR_INVALID_PARAM;
-	switch (type) {
-	case LOADCMD_LOAD_ALL:
-		ret = binary_manager_load_all();
-		break;
-	case LOADCMD_RELOAD:
-		if (argc <= 2) {
-			bmdbg("Invalid arguments for reloading, argc %d\n", argc);
-			break;
+	while (1) {
+		ret = sem_wait(&g_loading_list.sem);
+		if (ret != OK) {
+			bmdbg("sem_wait FAIL errno %d !!!!\n", errno);
+			continue;
+		}
+		ret = BINMGR_INVALID_PARAM;
```

Comment:
this ret is not used for returning. it is not needed to assign here.

Reply:
That value is used in log message for loading result, line:354